Lombiq.ChartJs.Tests.UI/Extensions/TestCaseUITestContextExtensions.cs

```diff
@@ -12,11 +12,21 @@
 using System.Linq;
 using System.Security.Cryptography;
 using System.Threading.Tasks;
-// This is because both of SixLabors.ImageSharp and System.Drawing are contains Image class,
-// but we want it from SixLabors.ImageSharp
+// We can't import the whole System.Drawing namespace because both
+// of SixLabors.ImageSharp and System.Drawing are contains Image class,
+// but we want it and some other from SixLabors.ImageSharp.
+// So we import only Bitmap from System.Drawing here.
 using Bitmap = System.Drawing.Bitmap;
 
+// The SixLabors.ImageSharp.Web[2.0.0] which is depends on SixLabors.ImageSharp[2.1.1]
+//  will be introduced in Orchard in the future.
+//  see: https://github.com/OrchardCMS/OrchardCore/pull/11585
+// When it will be done and we change the new version of OC in Lombiq.ChartJs.Samples,
+//  we should change from ImageSharpCompare[1.2.11]
+//  to Codeuctivity.ImageSharpCompare[2.0.46] in this project
```

Comment:
Are those version numbers in the square brackets? I haven't seen this kind of notation before, where does it come from? To be more obvious, just write "version 1.2.11" or "v1.2.11" instead.